packages/table/src/locator.ts

```diff
@@ -76,6 +78,27 @@ export class Locator implements ILocator {
         return max;
     }
 
+    public getTallestVisibleCellInColumn(columnIndex: number): number {
+        const cells = this.tableElement.querySelectorAll(`.${Classes.columnCellIndexClass(columnIndex)}`);
+        let max = 0;
+        for (let i = 0; i < cells.length; i++) {
+            const cellValue = (cells.item(i) as any).query(`.${Classes.TABLE_TRUNCATED_VALUE}`);
```

Comment:
For some reason the typings don't seem to think this thing has a "query" attached to it. Even though it does. This is what I wanted to ask about. If there's a better way to do this, please let me know.